client/src/constants.ts

```diff
@@ -23,3 +23,18 @@ export interface ItemData {
   amount: number;
   type: TransactionType;
 }
+
+export const months = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+] as const;
```

Comment:
'as const'? Maybe typo?

Reply:
Not a typo, in JavaScript, const actually can be changed in some ways. With typescript you can use "as const" to make it a true Const that cannot be changed.

Also, I forgot that it should be in all caps. MONTHS. It's just to emphasize that it is a true const.  Maybe you can make that change when you start working with it.

Reply:
ahhh Never seen this before! Good to know!